.github/workflows/benchmark.yml

```diff
@@ -21,7 +21,7 @@ on:
 
 jobs:
   run:
-    runs-on: buildjet-4vcpu-ubuntu-2204
+    runs-on: buildjet-4vcpu-ubuntu-2404
```

Comment:
these runners don't exist https://buildjet.com/for-github-actions/docs/runners/hardware which causes the CI to be stuck